src/unix/stream.c

```diff
@@ -769,18 +779,14 @@ static int uv__try_write(uv_stream_t* stream,
   if (send_handle != NULL) {
     int fd_to_send;
     struct msghdr msg;
-    struct cmsghdr *cmsg;
-    union {
-      char data[64];
-      struct cmsghdr alias;
-    } scratch;
+    union uv__cmsg cmsg;
 
     if (uv__is_closing(send_handle))
       return UV_EBADF;
 
     fd_to_send = uv__handle_fd((uv_handle_t*) send_handle);
 
-    memset(&scratch, 0, sizeof(scratch));
+    memset(&cmsg, 0, sizeof(cmsg));
```

Comment:
small nit: I think C++ compilers might have rejected this, though it was valid in C.
```suggestion
    memset(&cmsg.pad, 0, sizeof(cmsg));
```

Reply:
The past tense confuses me. It _is_ valid C, right? :-)